src/test/java/com/endava/service/IncomeExpenseServiceImplTest.java

```diff
@@ -193,15 +189,15 @@ public void testUpdateIncomeIncorrectDate() {
 
     @Test
     public void testViewIncomesOnCorrectDate() {
-        List<Income> incomes = new ArrayList<>( Arrays.asList(income1, income2, income3, income4) );
+        List<MoneyTransfer> moneyTransfers = new ArrayList<>( Arrays.asList(moneyTransfer, moneyTransfer1, moneyTransfer2, moneyTransfer3) );
         Date date = new Date(1489960800000L);
 
-        when(incomeDao.readAll()).thenReturn(incomes);
+        when(moneyTransferDao.readAll()).thenReturn(moneyTransfers);
 
-        List<Income> actualIncomes = incomeExpenseService.viewIncomesOn(date);
-        List<Income> expectedIncomes = new ArrayList<>( Arrays.asList(income1) );
+        List<MoneyTransfer> actualIncomes = incomeExpenseService.getMoneyTransferOn(date);
+        List<MoneyTransfer> expectedIncomes = new ArrayList<>( Arrays.asList(moneyTransfer) );
```

Comment:
You don't need new ArrayList here. Arrays.asList returns unmodifiable List, right what you need :)